README.md

```diff
@@ -1,23 +1,36 @@
 # Enarx
 
-## Introduction
-Enarx is an application deployment system enabling applications to run within Trusted Execution Environments (TEEs) without rewriting for particular platforms or SDKs. It handles attestation and delivery into a run-time “Keep” based on WebAssembly, offering developers a wide range of language choices for implementation. Enarx is CPU-architecture independent, enabling the same application code to be deployed across multiple targets, abstracting issues such as cross-compilation and differing attestation mechanisms between hardware vendors. Work is currently underway on AMD SEV and Intel SGX.
-
-We've known for a long time that we need encryption for data at rest and in transit: Enarx helps you do encryption for data as well as algorithms in use.
+## Introduction Enarx is an application deployment system enabling applications
+to run within Trusted Execution Environments (TEEs) without rewriting for
+particular platforms or SDKs. It handles attestation and delivery into a
+run-time “Keep” based on WebAssembly, offering developers a wide range of
+language choices for implementation. Enarx is CPU-architecture independent,
+enabling the same application code to be deployed across multiple targets,
+abstracting issues such as cross-compilation and differing attestation
+mechanisms between hardware vendors. Work is currently underway on AMD SEV and
+Intel SGX.
+
+We've known for a long time that we need encryption for data at rest and in
+transit: Enarx helps you do encryption for data as well as algorithms in use.
 
 [Read more](https://github.com/enarx/enarx/wiki/Enarx-Introduction)
 
 ## Project aim
-Create a way to create and run "private, fungible, serverless" applications using Trusted Execution Environments (TEEs). In other words, to provide a platform abstraction for TEEs.
-
-They should:
-
- * be confidential from the host and other workloads;
- * be able to run on
-   * multiple silicon architectures;
-   * multiple server types;
-   * public cloud or private cloud;
- * employ serverless deployment and execution mechanisms.
+The aim of the Enarx project is to create a way to allow easy deployment of
```

Comment:
```suggestion
Create a way to allow easy deployment of
```

We already specify that this is the project's aim; no need to say it twice.

Reply:
Don't entirely agree, "Project aim" is the name on the section. I'd argue using fully formed sentences after the title of the section to repeat the same idea is fine, the opposite is a bit too terse.